firefox-ios/Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewController.swift

```diff
@@ -71,33 +71,32 @@ class DefaultBrowserOnboardingViewController: UIViewController, OnViewDismissabl
     }
 
     private lazy var titleLabel: UILabel = .build { [weak self] label in
-        label.font = DefaultDynamicFontHelper.preferredBoldFont(withTextStyle: .title1,
-                                                                size: self?.titleFontSize ?? UX.titleSize)
```

Comment:
@darkCavalier11 Thanks for working on this, just following up - we can remove `titleFontSize` and associated constants since they are unused now.

Reply:
@cyndichin done!